CliClient/locales/id_ID.po

```diff
@@ -47,20 +47,20 @@ msgstr "Tidak ada notebook yang ditentukan."
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/app.js:136
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/cli-utils.js:160
 msgid "Y"
-msgstr "Ya"
+msgstr "Y"
 
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/app.js:136
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/cli-utils.js:160
 msgid "n"
-msgstr "tidak"
+msgstr "t"
 
 #: /Users/tessus/data/work/joplin/Tools/../CliClient/app/app.js:136
 msgid "N"
-msgstr "Tidak"
+msgstr "N"
```

Comment:
Thanks for the change. One last question:

Is there a reason why `n -> t` and `N -> N`? I would have imagined due to consistency it would be `N -> T`.

Reply:
i forgot that, i'll change to N -> T